client/my-sites/domains/domain-management/components/inbound-transfer-verification/index.jsx

```diff
@@ -68,8 +69,7 @@ class InboundTransferEmailVerificationCard extends React.Component {
 						components: {
 							learnMoreLink: (
 								<a
-									href="http://support.wordpress.com"
-									target="_blank"
+									href={ support.INCOMING_DOMAIN_TRANSFER }
```

Comment:
@danhauk, do we need a different link here?

Reply:
@aidvu We'll have a support article specifically for the various statuses of the transfer. The slug will be `/incoming-domain-transfer/status-and-failed-transfers/`. I'm not sure of the exact hash anchors yet, but the statuses we'll have on that page are:

* Pending Confirmation (and how to resend the email)
* Pending Transfer
* Transfer Failed
* Cancel Transfer